public/coffee/ide/labels/LabelsManager.coffee

```diff
@@ -0,0 +1,59 @@
+define [
```

Comment:
One option is for this to go under public/coffee/ide/editor/directives/aceEditor like the spell check and other editor extensions.